recipes/sdl2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "SDL2" %}
+{% set version = "2.0.4" %}
+{% set sha256_targz = "da55e540bf6331824153805d58b590a29c39d2d506c6d02fa409aedeab21174b" %}
+{% set sha256_zip = "08ba1de8bb4236d3aa1a94168b6e1a4f286bd21ca3437c11689d4e55f3a9cc89" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz  # [unix]
+  url:  https://www.libsdl.org/release/{{ name }}-{{ version }}.tar.gz  # [unix]
+  sha256: {{ sha256_targz }}  # [unix]
+  fn:  {{ name }}-{{ version }}.zip  # [win]
+  url:  https://www.libsdl.org/release/{{ name }}-{{ version }}.zip  # [win]
+  sha256: {{ sha256_zip }}  # [win]
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - cmake  # [win]
+    - autoconf  # [unix]
+
```

Comment:
The only thing is that sdl-config is not available in Windows as an executable, so that won't work there. I can add this and then add the # [unix] selector?